src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/twig.html.twig

```diff
@@ -25,6 +25,15 @@
             <b>Macro Calls</b>
             <span class="sf-toolbar-status">{{ collector.macrocount }}</span>
         </div>
+        <div class="sf-toolbar-info-piece">
+            <b>Template</b>
+            <span>
+                {% for template, count in collector.templates|slice(0,1) %}
+                    {%- set file = collector.templatePaths[template]|default(false) -%}
+                    {{ file|file_relative|default(file) }}
```

Comment:
what happens here if the path is `false` ? shouldn't it be `|default(template)` instead ?
And why showing a file path rather than a template name ?

Reply:
The idea behind showing the file path rther than the template name is for cases when you have the following templates tree :

/templates
----user/
--------index.html.twig
--------view.html.twig
----customer/
--------index.html.twig
--------view.html.twig

Just showing view.html.twig might not be sufficient in this case.
Sometimes when you take an existing project for a client you don't really have the choice to use those kinds of tree

Reply:
the template name is **not** `view.html.twig`. I'm not talking about the file name, but about the **template name** (was you give to Twig to render it)

Reply:
Hmm sorry I misunderstood you. Yes actually it would be better to show the template name.